scripts/filemanager.js

```diff
@@ -9,8 +9,34 @@
  *	@author		Pavel Solomienko <https://github.com/servocoder/>
  *	@copyright	Authors
  */
-
+ 
 (function($) {
+  
+  // keep variables in non global object
+var fm = {};
```

Comment:
I have got your point. Indeed it's more flexible to change context if you have all variables under the context of a single object. I don't care of a reference chain lehgth actually, but I'm worry a bit not to miss any variable being added to `vm` object. Well, let's postpone the decision on that.